src/styles/blocks/header.scss

```diff
@@ -0,0 +1,58 @@
+.header {
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  margin: 10px 0;
+}
+
+.nav {
```

Comment:
For each BEM block you should create a separate style file. Check the 8th point from the [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)